src/api/tokenAddresses.ts

```diff
@@ -0,0 +1,33 @@
+// TODO: refactor and get these from the same file instead of repeating it everywhere
+export enum Network {
```

Comment:
They are also in `initWeb3.js` if you'd like to use that, but maybe it makes sense to move it to a dedicated `constants.js` in `utils`

Reply:
Would you mind if I do it as a `ts` file instead, to have one less thing to refactor later on?
I mean, assuming it would work on `.js` files, have to try it out.

Reply:
Sure, `.ts` is fine too